pkgs/development/tools/build-managers/buildbot/default.nix

```diff
@@ -1,76 +1,80 @@
-{ stdenv, lib, fetchurl, coreutils, openssh, buildbot-worker, makeWrapper,
-  pythonPackages, gnused, plugins ? [] }:
+{ stdenv, lib, openssh, buildbot-worker, pythonPackages, runCommand, makeWrapper }:
 
-pythonPackages.buildPythonApplication (rec {
-  name = "${pname}-${version}";
-  pname = "buildbot";
-  version = "0.9.4";
-  src = fetchurl {
-    url = "mirror://pypi/b/${pname}/${name}.tar.gz";
-    sha256 = "0wklrn4fszac9wi8zw3vbsznwyff6y57cz0i81zvh46skb6n3086";
-  };
+let
+  withPlugins = plugins: runCommand "wrapped-${package.name}" { buildInputs = [ makeWrapper ]; } ''
+    makeWrapper ${package}/bin/buildbot $out/bin/buildbot \
+      --set PYTHONPATH ${lib.makeSearchPathOutput "lib" pythonPackages.python.sitePackages plugins}
```

Comment:
Yep. Earlier there was a PR to change `--prefix PYTHONPATH` to `--set PYTHONPATH`. I didn't want that because `PYTHONPATH` is still a Python feature. That's the reason we now patch scripts. Now you break that feature ;-)